react/features/base/tracks/middleware.js

```diff
@@ -176,6 +197,63 @@ MiddlewareRegistry.register(store => next => action => {
     return next(action);
 });
 
+/**
+ * Gets the participant ID of the given jitsi track.
+ *
+ * @param {Store} store - The redux store.
+ * @param {JitsiTrack} jitsiTrack - The jitsi media track.
+ * @private
+ * @returns {string} - The participant ID.
+ */
+function _getJitsiTrackParticipantId(store, jitsiTrack) {
+    return jitsiTrack.isLocal()
```

Comment:
I'm not sure you need this function at all but if you do maybe you can use `getTrackByJitsiTrack` and from there you can get the participantID. WDYT?